SparkPost/2web2ui
# FE-868: Checkbox component wrapper

### What Changed
 - Added checkbox component wrapper

### How To Test
 - Check that sections of the app with checkboxes are working correctly (like API keys page)
 - Check that CheckboxWrapper component is working correctly (redux form)
 - Check that there are no major default visual errors

### To Do
